src/fielddef.ts

```diff
@@ -339,16 +339,18 @@ export function isScaleFieldDef<F>(channelDef: ChannelDef<F>): channelDef is Sca
 }
 
 export interface FieldRefOption {
-  /** exclude bin, aggregate, timeUnit */
+  /** Exclude bin, aggregate, timeUnit */
   nofn?: boolean;
   /** Wrap the field with datum or parent (e.g., datum['...'] for Vega Expression */
   expr?: 'datum' | 'parent';
-  /** prepend fn with custom function prefix */
+  /** Prepend fn with custom function prefix */
   prefix?: string;
-  /** append suffix to the field ref for bin (default='start') */
+  /** Append suffix to the field ref for bin (default='start') */
   binSuffix?: 'end' | 'range' | 'mid';
-  /** append suffix to the field ref (general) */
+  /** Append suffix to the field ref (general) */
   suffix?: string;
+  /** Use the field name for `as` in a transform. */
```

Comment:
Add a comment how this affect the output.

Reply:
Done